R/getREMINDTemplateVariables.R

```diff
@@ -0,0 +1,20 @@
+#' Retrieves all REMIND variables potentially used in mappings to project variables
+#'
+#' @md
+#' @author Falk Benke
+#' @importFrom dplyr %>%
+#' @export
+getREMINDTemplateVariables <- function() {
```

Comment:
Depending on how we want to design the tests, this could become a function only returning the variables of a specific template. 

I.e. if there is a test (and possibly a separate gdx) for each supported project, then this function should also be per project template.